.storybook/stories/tooltip.stories.mdx

```diff
@@ -10,9 +10,31 @@ import { Tooltip } from "../misc";
     component={TooltipWrapper}
 />
 
+<ArgsTable of={TooltipWrapper} />
+
+## Manual Position w/ Happy Path (enough room)
 <Canvas>
-    <Story name="Tooltip Wrapper">
+    <Story name="Default">
         <div className="story__container tooltip-story">
+            <TooltipWrapper
+                icon="info"
+                tooltipComponent={<Tooltip />}
+                wide={boolean("wide", false)}
+                tooltipPosition={text("tooltipPosition", "left")}
+                controlledProps={object("controlledTooltip", {
+                    isControlled: false,
+                    isVisible: false
+                })} />
+        </div>
+    </Story>
+</Canvas>
+
+When positioning the tooltip right/left, if there isn't enough room, it chooses the position for you.
```

Comment:
It's unclear whether this refers to the Default or Dynamic variations. I _think_ it refers to the two dynamic Stories, in which case I'd recommend adding another header to the markdown for Smart / Dynamic tooltips.

Reply:
Its actually always dynamic... So there's no prop that says okay be smart it just always is smart. Kind of implies that it is a "variation" by putting it on the sidebar but I wanted to show it off somehow?